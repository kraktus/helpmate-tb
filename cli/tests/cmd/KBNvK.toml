args = "--no-time generate -m \"KBNvK\" --tb-dir ../table"
bin.name = "helpmate_tb_cli"
