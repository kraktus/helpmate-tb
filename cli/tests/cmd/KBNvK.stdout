[INFO  helpmate_tb_cli::generate] Building KBNvK with winner: white
[DEBUG helpmate_tb::generation] all_pos_vec capacity: 4288788
[DEBUG helpmate_tb::generation] all_pos_vec capacity: 1891839 after shrinking
[DEBUG helpmate_tb::generation] nb pos 1891839
[DEBUG helpmate_tb::generation] counter 3813096
[DEBUG helpmate_tb::generation] nb White mate 58
[DEBUG helpmate_tb::generation] nb Black mates 0
[DEBUG helpmate_tb_cli::explore] From White perspective, win: 3064718, draw: 2748, lost: 0, unkown: 0
[DEBUG helpmate_tb_cli::explore] Index density = 81%
[DEBUG helpmate_tb_cli::explore] Win(0), 58
[DEBUG helpmate_tb_cli::explore] Win(1), 230
[DEBUG helpmate_tb_cli::explore] Win(2), 417
[DEBUG helpmate_tb_cli::explore] Win(3), 2538
[DEBUG helpmate_tb_cli::explore] Win(4), 8084
[DEBUG helpmate_tb_cli::explore] Win(5), 48104
[DEBUG helpmate_tb_cli::explore] Win(6), 109912
[DEBUG helpmate_tb_cli::explore] Win(7), 285316
[DEBUG helpmate_tb_cli::explore] Win(8), 523280
[DEBUG helpmate_tb_cli::explore] Win(9), 594013
[DEBUG helpmate_tb_cli::explore] Win(10), 752753
[DEBUG helpmate_tb_cli::explore] Win(11), 362230
[DEBUG helpmate_tb_cli::explore] Win(12), 283057
[DEBUG helpmate_tb_cli::explore] Win(13), 64143
[DEBUG helpmate_tb_cli::explore] Win(14), 27498
[DEBUG helpmate_tb_cli::explore] Win(15), 2989
[DEBUG helpmate_tb_cli::explore] Win(16), 96
[INFO  helpmate_tb_cli::generate] Building KBNvK with winner: black
[DEBUG helpmate_tb::generation] all_pos_vec capacity: 4288788
[DEBUG helpmate_tb::generation] all_pos_vec capacity: 1891839 after shrinking
[DEBUG helpmate_tb::generation] nb pos 1891839
[DEBUG helpmate_tb::generation] counter 3813096
[DEBUG helpmate_tb::generation] nb Black stalemate/capture resulting in draw 1611
[DEBUG helpmate_tb::generation] nb White mates 58
[DEBUG helpmate_tb_cli::explore] From Black perspective, win: 0, draw: 3067408, lost: 58, unkown: 0
[DEBUG helpmate_tb_cli::explore] Index density = 81%
[DEBUG helpmate_tb_cli::explore] Lose(0), 58
